Import/Esri/ArcGISRuntime/Toolkit/Controls/QmlApi/TimeSliderController.qml

```diff
@@ -18,26 +18,84 @@ import QtQuick 2.4
 import Esri.ArcGISRuntime 100.3
 
 /*!
-    \qmltype TimeSlider
+    \qmltype TimeSliderController
     \ingroup ArcGISQtToolkit
     \ingroup ArcGISQtToolkitQmlApi
     \inqmlmodule Esri.ArcGISRuntime.Toolkit.Controls
     \since Esri.ArcGISRutime 100.3
-    \brief
+    \brief The controller for the Time Slider tool.
+
+    The controller presents the temporal range of the data as a number of steps.
```

Comment:
Is the controller something someone will want to create on their own, or does the TimeSlider create it for you, and you just access it through the TimeSlider? If so, is there a property to access it from TimeSlider?